CHANGELOG.md

```diff
@@ -1,6 +1,12 @@
-## Unreleased
+## v0.15.0
 
-* Add changes here
+* Plugin release milestone
+
+## v0.14.1
+
+### BUG FIXES:
+
+* Fix bug where updates to config would fail is password isn't provided
```

Comment:
```suggestion
* Fix bug where updates to config would fail if password isn't provided
```

We also usually link to the PR similar to other entries below.

Reply:
Good typo catch. I'll add that too.